src/Infrastructure.VS/FolderWorkspaceService.cs

```diff
@@ -63,12 +63,12 @@ public string FindRootDirectory()
             return solutionInfoProvider.GetSolutionDirectory();
         }
 
-        public IEnumerable<string> ListFiles()
+        public ICollection<string> ListFiles()
         {
             var root = FindRootDirectory();
 
             return root is not null ?
```

Comment:
We need to fix the coding style of this statement (not now)

Reply:
You mean the `?` at the end of the first line rather than at the beginning of the second? Ugras needs to fix his editor settings ;-)

Reply:
Done